### PR TITLE
feat: add chronix tests

### PR DESCRIPTION
**Pull request by eddort**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,8 +34,11 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config jest-e2e.json",
-    "typechain": "typechain --target=ethers-v5 --out-dir ./src/generated ./src/staking-router-modules/contracts/abi/*.json"
+    "test:e2e": "jest --config jest-e2e.json && chronix test",
```
Comment (Amuhar) [inline package.json:37]: Could you explain please what `chronix test`  command do ?
What is the difference with `test:e2e:onchain` 
Why we didnt add action in gh workflow for test:e2e:onchain ?

Comment (Amuhar) [reply package.json:37]:
```
yarn test:e2e:onchain
yarn run v1.22.15
$ jest --config  jest-e2e-onchain.json
 FAIL  src/app/simple-dvt.e2e-chain.ts
  ● Simple DVT › initial state created

    FetchError: request to http://localhost:8001/v1/ready? failed, reason: connect ECONNREFUSED 127.0.0.1:8001

      at ClientRequest.<anonymous> (node_modules/chronix/node_modules/node-fetch/lib/index.js:1491:11)

  ● Simple DVT › update keys api keys

    FetchError: request to http://localhost:8001/v1/ready? failed, reason: connect ECONNREFUSED 127.0.0.1:8001
```

Comment (Amuhar) [reply package.json:37]: What am I doing wrong?

Comment (Amuhar) [reply package.json:37]: I don't like that `test:e2e:onchain` doesn't work separately and work only as `chronix test`

**Comment by Amuhar** [comment]
The tests with Chronix look good! However, is it enough to check the number of operators and keys, or should we also verify the operator IDs, the number of used/unused keys?

File: package.json (hunk 1 above)
Comment (eddort) [reply package.json:37]: Yeah, it's looking a little confusing at the moment.
To work with Chronix you need to run it in a separate process. This is not very convenient, so I made a special `chronix test` command. This command starts chronix and then executes the command from chronix.config.js. In our case, it's `test:e2e:onchain`.
It turns out that by calling chronix test the following happens: chronix starts -> jest test call -> test process terminates -> chronix process terminates.
In this way interaction with tests is greatly simplified and there is no need to start a separate process on its own.

**Review by Amuhar — APPROVED** [review]

**Review by Amuhar — APPROVED** [review]

**Review by Amuhar — APPROVED** [review]

**Pull request merged**